ipatests/test_webui/test_navigation.py

```diff
@@ -106,12 +106,11 @@ def test_menu_navigation(self):
 
         # Identity
         # don't start by users (default)
-        self.navigate_by_menu('identity/group', False)
+        self.navigate_by_menu('identity/group_search', False)
         self.navigate_by_menu('identity/user_search', False)
         self.navigate_by_menu('identity/host', False)
-        self.navigate_by_menu('identity/hostgroup', False)
```

Comment:
I'm not sure if removing the hostgroup completely is the right thing. The test failure was caused by: 070bc48dd6c9bce32caa0f0f2de8d44b4e5bbbb1 We might want to do something like:

```
self.navigate_by_menu('identity/group_search', False)
self.navigate_by_sidebar('Host Groups', False)
self.navigate_by_sidebar('Netgroups, False)
```
I'm not sure if we a methog like `navigate_by_sidebar` implemented though.

Reply:
good catch. We don't have `navigate_by_sidebar` but we do have `switch_to_facet` that looks for a tab and if it doesn't exist, it looks for a link in the sidebar.